aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -150,6 +150,11 @@ class CAFFE2_API Dispatcher final {
   // it for a bit until the real schema turns up
   RegistrationHandleRAII registerImpl(OperatorName op_name, c10::optional<DispatchKey> dispatch_key, KernelFunction kernel, std::unique_ptr<FunctionSchema> inferred_function_schema, std::string debug);
 
+  /**
+   * Register a new operator by name.
```

Comment:
Here it would be really helpful to have an explanation about under what circumstances this should be used. A reference to the relevant issue would be good.